src/cf-forms/usage.md

```diff
@@ -750,9 +771,7 @@ typically to clear the input.
 
 <div class="m-btn-inside-input">
     <input type="text"
-        value="This is some really long text to make sure that the button
-               doesn't overlap the content in such a way that this input
-               becomes unusable."
+        value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable."
```

Comment:
Is it no longer possible to break this under 80char line lengths?

Reply:
I don't think this ever worked correctly, check out https://cfpb.github.io/capital-framework/components/cf-forms/

The whitespace ends up in the output
![screen shot 2018-01-02 at 2 05 45 pm](https://user-images.githubusercontent.com/704760/34496065-12ea89e8-efc6-11e7-99e8-89a266e04251.png)

Reply:
Oh, hahaha, glad you caught it